tests/e2e/app/android/app/proguard-rules.pro

```diff
@@ -0,0 +1,10 @@
+# Add project specific ProGuard rules here.
```

Comment:
Can we delete this file, along with any other boilerplate ones that aren't used?